### ISSUE TITLE
Implement support of multilingual interface

### ISSUE THREAD

**Issue by sadko4u** [sadko4u/lsp-plugins#65]
Current state of development requires to implement multilingual support in the system.

Translation files should be stored as set of JSON bundles, each bundle can be addressed as dot-separated set of identifiers and have optional named parameters in the text.

The final message text should be rendered by using special object that will construct it from language label, bundle key and set of additional named parameters. If there is no language bundle for the selected language, the bundle for the default language should be selected.

All JSON files stored in the project should be built-in into the final binaries to avoid additional resource lookups and possible resource misses.

**Comment by trebmuh** [comment]
As said on #63, ping me when done, then I'll make a French translation.

**Comment by sadko4u** [comment]
Merged multilingual support into **devel** branch: 9b67912ff5ee267226c571770c23be8e515ad690

@trebmuh you can now make a french translation.

@CrocoDuckoDucks you can do italian translation for the UI, too.

Please make branches from **devel** branch or checkout **github-issue-65** branch for further changes.

Location of files that contain translations:
```
lsp-plugins/res/i18n
```
To create your own translations, you may copy files from **us** directory into the new directory with your language identifier. Then, just edit contents of files.
The usual structure of files is the following:
```javascript
{
	"comp_delay": {
		"distance": "Distance",
		"samples": "Samples",
		"time": "Time"
	},
	
	"compressor": {
		"down_ward": "Down",
		"downward": "Downward",
		"up_ward": "Up",
		"upward": "Upward"
	}
        //... etc ...
}
```

Each element is accessible by the full path which is set of dot-separated keys where the first key is a JSON file name. So, for example, if we stored the JSON data above in the 'lists.json' file, the path:
```
lists.compressor.downward
```
Will be translated into text:
```
Downward
```
The language is split into current language and default language (**default** directory). If there is no key in current language, lookup in the default language registry will be performed. If there is no entry in default language registry, the lookup key will be displayed instead of the localized text.

To make your language selectable in the UI, add the key with language description to **default.lang.target** path.

To find location where the localized value is used, you can simply grep all XML files for full or partial path of this value:
```bash
grep -R 'labels.fft.envelope' res/ui/*

res/ui/analyzer/spectrum/x1.xml:                                        <label text="labels.fft.envelope" />
res/ui/analyzer/spectrum/x8.xml:                                        <label text="labels.fft.envelope" />
res/ui/analyzer/spectrum/x16.xml:                                       <label text="labels.fft.envelope" />
res/ui/analyzer/spectrum/x12.xml:                                       <label text="labels.fft.envelope" />
res/ui/analyzer/spectrum/x4.xml:                                        <label text="labels.fft.envelope" />
res/ui/analyzer/spectrum/x2.xml:                                                <label text="labels.fft.envelope" />
```

Changing language of the UI is possible by clicking on the **LSP** shield and selecting language from corresponding menu item. All translations should be applied immediately.

**Comment by sadko4u** [comment]
Removed branch **github-issue-65**. Please use **devel** branch for additional localizations.

**Comment by sadko4u** [comment]
Changes available in 1.1.14 release!
